python/tests/integration/arcticdb/version_store/test_basic_version_store.py

```diff
@@ -1947,6 +1947,13 @@ def test_get_index(object_version_store):
     assert idx.iloc[0]["version_id"] == 0
 
 
+def test_read_empty_index(lmdb_version_store_v1):
```

Comment:
Test for v2 encoding?

Reply:
This change sits far above the encoding layer, not everything needs to be tested with both